### ISSUE TITLE
Command 'View On Map' resulted in an error

### ISSUE THREAD

**Issue by csaper** [RE0x2A/geojson.io-for-vscode#24]
Hey Dan - 

Just added the geoJSON.IO extension to VSCode. I opened a geojson file and ran the command `View on Map` per the usage instructions and received the following error:

![geojsonio-vscode-extension-error](https://user-images.githubusercontent.com/316563/172939110-6d7543e5-a22d-425c-8a53-0cf8944c4a08.png)

<details>
<summary>Here is the output of the Extension Host Log:</summary>

```
[2022-06-09 16:21:37.784] [exthost] [error] TypeError: Cannot read properties of undefined (reading 'path')
    at ~/.vscode/extensions/swallow.geojson-io-for-vscode-0.4.1/out/src/extension.js:26:17
    at o._executeContributedCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:62521)
    at o.$executeContributedCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:63225)
    at s._doInvokeHandler (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:86:13828)
    at s._invokeHandler (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:86:13512)
    at s._receiveRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:86:12173)
    at s._receiveOneMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:86:10843)
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:86:8949
    at g.invoke (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:145)
    at b.deliver (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:2265)
    at v.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:1843)
    at a.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:66:19025)
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:34863
    at g.invoke (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:145)
    at b.deliver (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:2265)
    at v.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:1843)
    at a.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:66:19025)
    at r._receiveMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:66:23606)
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:66:21140
    at g.invoke (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:145)
    at b.deliver (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:2265)
    at v.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:58:1843)
    at v.acceptChunk (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:66:15856)
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:66:14986
    at Socket.R (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:13798)
    at Socket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at Socket.Readable.push (node:internal/streams/readable:228:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:199:23) geojson-io-for-vscode.openMap {"value":"swallow.geojson-io-for-vscode","_lower":"swallow.geojson-io-for-vscode"}
```
</details>

**Stack**
VSCode v1.68.0
macOS v12.3.1
GeoJSON.IO for VSCode v0.4.1

Let me know if I can provide further info to support debugging.

Craig

**Comment by anders86755** [comment]
I got the same error. 

VSCode V1.68
Windows 10
GeoJson.IO for VSCode -- all version

**Comment by 42arch** [comment]
Hey, Craig and Anders, thx for your report. Yeah, It's definitely a bug, I didn't notice that before. I will fix it in the next version, Before that, you can right-click on your file tab and select the "View on Map" option. Just like this, they have same effects:

![new json - Visual Studio Code 2022_6_10 12_49_34](https://user-images.githubusercontent.com/20656708/172992845-cfc848ca-346e-4f65-942b-500ac8910c41.png)

**Comment by anders86755** [comment]
the same result.
My right-click menu is different from yours. It doesn't have "view on map" option. I only have one plugin installed,-- your plugin. When I enter "command palette" and select "view on map" command, I got the same error. 
I tested many geojson files. These files are generated on geojson.io website.
![20220610141125](https://user-images.githubusercontent.com/40052500/173003916-3072fd06-f72a-417a-9394-76881224afc9.png)
![20220610141156](https://user-images.githubusercontent.com/40052500/173003929-8811bdc1-6fc4-4105-a4a9-b1905f6f7298.png)
![20220610141216](https://user-images.githubusercontent.com/40052500/173003944-dc37fbde-02ce-4e07-91c2-1003dbfd5819.png)
